src/http/AuthHttpClient.js

```diff
@@ -504,6 +504,28 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Gets the balances of an array of accounts.
+     *
+     * @param {Array} accountIds - array of accountIds
+     * @param {string} keyLevel - key level
+     * @return {Object} response - response to the API call
+     */
+    async getBalances(accountIds, keyLevel) {
+        this.useKeyLevel(keyLevel);
+        var url = '/accounts/balance?';
+
+        for (var i = 0; i < accountIds.length; ++i) {
```

Comment:
Might be able to impress your functional programmer friends via accountIds.map(...)

Reply:
done